backend/pkg/api/data_access/notifications.go

```diff
@@ -77,11 +100,534 @@ func (d *DataAccessService) UpdateNotificationSettingsClients(ctx context.Contex
+	// }
+	// defer utils.Rollback(tx)
+
+	// eventFilter := fmt.Sprintf("%s:%d:%d", AccountDashboardEventPrefix, dashboardId, groupId)
+
+	// d.AddOrRemoveEvent(&eventsToInsert, &eventsToDelete, settings.IsIncomingTransactionsSubscribed, userId, string(types.IncomingTransactionEventName), eventFilter, epoch, 0)
+	// d.AddOrRemoveEvent(&eventsToInsert, &eventsToDelete, settings.IsOutgoingTransactionsSubscribed, userId, string(types.OutgoingTransactionEventName), eventFilter, epoch, 0)
+	// d.AddOrRemoveEvent(&eventsToInsert, &eventsToDelete, settings.IsERC20TokenTransfersSubscribed, userId, string(types.ERC20TokenTransferEventName), eventFilter, epoch, settings.ERC20TokenTransfersValueThreshold)
+	// d.AddOrRemoveEvent(&eventsToInsert, &eventsToDelete, settings.IsERC721TokenTransfersSubscribed, userId, string(types.ERC721TokenTransferEventName), eventFilter, epoch, 0)
+	// d.AddOrRemoveEvent(&eventsToInsert, &eventsToDelete, settings.IsERC1155TokenTransfersSubscribed, userId, string(types.ERC1155TokenTransferEventName), eventFilter, epoch, 0)
+
+	// // Insert all the events or update the threshold if they already exist
+	// if len(eventsToInsert) > 0 {
+	// 	insertDs := goqu.Dialect("postgres").
+	// 		Insert("users_subscriptions").
+	// 		Cols("user_id", "event_name", "event_filter", "created_ts", "created_epoch", "event_threshold").
+	// 		Rows(eventsToInsert).
+	// 		OnConflict(goqu.DoUpdate(
+	// 			"user_id, event_name, event_filter",
+	// 			goqu.Record{"event_threshold": goqu.L("EXCLUDED.event_threshold")},
+	// 		))
+
+	// 	query, args, err := insertDs.Prepared(true).ToSQL()
+	// 	if err != nil {
+	// 		return fmt.Errorf("error preparing query: %v", err)
+	// 	}
+
+	// 	_, err = tx.ExecContext(ctx, query, args...)
+	// 	if err != nil {
+	// 		return err
+	// 	}
+	// }
+
+	// // Delete all the events
+	// if len(eventsToDelete) > 0 {
+	// 	deleteDs := goqu.Dialect("postgres").
+	// 		Delete("users_subscriptions").
+	// 		Where(goqu.Or(eventsToDelete...))
+
+	// 	query, args, err := deleteDs.Prepared(true).ToSQL()
+	// 	if err != nil {
+	// 		return fmt.Errorf("error preparing query: %v", err)
+	// 	}
+
+	// 	_, err = tx.ExecContext(ctx, query, args...)
+	// 	if err != nil {
+	// 		return err
+	// 	}
+	// }
+
+	// err = tx.Commit()
+	// if err != nil {
+	// 	return fmt.Errorf("error committing tx to update validator dashboard notification settings: %w", err)
+	// }
+
+	// // Set non-event settings
+	// _, err = d.alloyWriter.ExecContext(ctx, `
+	// 	UPDATE users_acc_dashboards_groups
+	// 	SET
+	// 		webhook_target = NULLIF($1, ''),
+	// 		webhook_format = CASE WHEN $2 THEN $3 ELSE NULL END,
+	// 		ignore_spam_transactions = $4,
+	// 		subscribed_chain_ids = $5
+	// 	WHERE dashboard_id = $6 AND id = $7`, settings.WebhookUrl, settings.IsWebhookDiscordEnabled, DiscordWebhookFormat, settings.IsIgnoreSpamTransactionsEnabled, settings.SubscribedChainIds, dashboardId, groupId)
+	// if err != nil {
+	// 	return err
+	// }
+
+	return nil
```

Comment:
```suggestion
	return d.dummy.UpdateNotificationSettingsAccountDashboard(ctx, userId, dashboardId, groupId, settings)
```

Reply:
Right, it should remain as before. Fixed.